content/cloud-docs/api-docs/providers.mdx

```diff
@@ -254,79 +312,411 @@ curl \
+
+`DELETE /organizations/:organization_name/registry-providers/:registry_name/:namespace/:name/versions/:provider_version`
 
 ### Parameters
 
-| Parameter            | Description                                                                                                                                                                                                  |
-| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `:organization_name` | The name of the organization to delete a provider from. The organization must already exist, and the token authenticating the API request must belong to the "owners" team or a member of the "owners" team. |
-| `:namespace`         | The namespace of the provider that will be deleted.                                                                                                                                                          |
-| `:name`              | The name of the provider that will be deleted.                                                                                                                                                               |
-| `:registry_name`     | `public`                                                                                                                                                                                                     |
+| Parameter            | Description                                                                                                                                                                                                          |
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization to delete a provider version from. The organization must already exist, and the token authenticating the API request must belong to the "owners" team or a member of the "owners" team. |
+| `:namespace`         | The namespace of the provider for which the version is being deleted. For private providers this is the same as the `:organization_name` parameter                                                                  |
+| `:name`              | The name of the provider for which the version is being destroyed                                                                                                                                                    |
+| `:version`           | The version for the provider that will be deleted                                                                                                                                                                    |
```

Comment:
Will the platforms attached to the version also be deleted? I assume so and maybe it's safe to assume folks would assume that? Or is it safer to note that the version for the provider along with the versions corresponding platforms that is being destroyed?